Automattic/pocket-casts-android
# Bookmarks: Add headphone controls settings screen

## Description

This adds bookmarks headphone controls settings screen. 
##### Just saves the settings, actual headphone actions are not implemented.

## Testing Instructions
1. Install debug build
2. Go to `Profile` -> ⚙️  and tap the `Headphone controls` menu
3. ✅ Verify the view appears and looks good
4. ✅ Verify the default values are: 
    - `Previous action`: `Skip back`
    - `Next action`: `Skip forward`
5. Tap on the `Previous action` item
6. ✅ Verify the options are:
    - `Skip back`
    - `Previous chapter`
    - `Add bookmark`
9. Tap the `Previous chapter`  item
10. ✅ Verify the secondary label reflects the change
11. Tap back to close the view, then reopen it
12. ✅ Verify the setting is persisted
13. Tap the `Next action` item
14. ✅ Verify the options are:
    - `Skip forward`
    - `Next chapter`
    - `Add bookmark`
15. Tap the `Next chapter` option
16. ✅ Verify the label changes, and the value is persisted
14. Tap either option and select `Add bookmark`
15. ✅ Verify the `Bookmark confirmation sound` row appears
17. ✅ Verify the value is persisted again
18. Tap the same action, and select anything but A`dd bookmark`
19. ✅ Verify the `Bookmark confirmation sound` row is hidden
20. Repeat the steps with the other section
21. With the `Bookmark confirmation sound` row visible, tap the Switch and ✅ verify the value is persisted
22. Tap the `Bookmark confirmation sound` row and ✅ verify the value is toggled

Disabled feature flag

1. Go to `Profile` -> ⚙️ -> `Beta features` and toggle off the `Bookmarks` feature
2. Restart the app 
3. Go to `Profile` -> ⚙️ 
4. ✅ Notice that `Headphone controls` menu is hidden

## Screenshots or Screencast 

![out](https://github.com/Automattic/pocket-casts-android/assets/1405144/e30c4a95-6510-4bfe-8ba7-6e99fa925385)

## Checklist
 
#### I have tested any UI changes...
